### PR TITLE
fix: update migration entry points

### LINKED ISSUE #447: Entry Points of all migrations are wrong
When the migrations were moved out of `ngxtension`, each entryPoint was not updated in the respective markdown file.  I believe this is what causes the 404 when you click the GitHub `code` button.

### PR DESCRIPTION
**Pull request by jdegand**
Closes #447.

### CODE REVIEW

**Review by nartc — APPROVED** [review]

**Review by nartc — CHANGES_REQUESTED** [review]
Actually, this wasn't the cause. Check the `PageTitle` component and update the `sourceCodeUrl` there.

**Comment by jdegand** [comment]
Change this `https://github.com/nartc/ngxtension-platform/tree/main/libs/ngxtension/${entryPoint}` to `https://github.com/nartc/ngxtension-platform/tree/main/libs/${entryPoint}` and add folder to all other entry points ?

I think `bundleJsUrl` will also have to be updated as well from `https://deno.bundlejs.com?q=ngxtension/${entryPoint}&treeshake=` to `https://deno.bundlejs.com?q=${entryPoint}&treeshake=`.

**Comment by nartc** [comment]
@jdegand 
- Change `https://github.com/nartc/ngxtension-platform/tree/main/libs/ngxtension/${entryPoint}` to `https://github.com/ngxtension/ngxtension-platform/tree/main/libs/ngxtension/${entryPoint}`
- The `bundleJsUrl` is fine because it reaches out to npmjs instead of repo's file structure.

**Comment by jdegand** [comment]
I don't this will work `https://github.com/ngxtension/ngxtension-platform/tree/main/libs/ngxtension/${entryPoint}` because the migration files are not inside the libs/ngxtension file.  They are inside the plugin folder.  

Other issues I noticed: `Gesture` vs `Gestures` file mismatch, `if-validator` does not have a readme file.

File: docs/src/components/PageTitle.astro [hunk 1]
```diff
@@ -5,8 +5,8 @@ import { Icon } from '@astrojs/starlight/components';
 
 const badge = Astro.props.entry.data.badge;
 const entryPoint = Astro.props.entry.data.entryPoint;
-const bundleJsUrl = `https://deno.bundlejs.com?q=ngxtension/${entryPoint}&treeshake=[*]&config={%22esbuild%22:{%22external%22:[%22rxjs%22,%22@angular/core%22,%22@angular/common%22,%22@angular/forms%22,%22@angular/router%22]}}`;
-const sourceCodeUrl = `https://github.com/nartc/ngxtension-platform/tree/main/libs/ngxtension/${entryPoint}`;
+const bundleJsUrl = `https://deno.bundlejs.com?q=${entryPoint}&treeshake=[*]&config={%22esbuild%22:{%22external%22:[%22rxjs%22,%22@angular/core%22,%22@angular/common%22,%22@angular/forms%22,%22@angular/router%22]}}`;
+const sourceCodeUrl = `https://github.com/nartc/ngxtension-platform/tree/main/libs/${entryPoint}`;
```
Comment (nartc) [inline docs/src/components/PageTitle.astro:9]:
```suggestion
const sourceCodeUrl = `https://github.com/ngxtension/ngxtension-platform/tree/main/libs/${entryPoint}`;
```

**Comment by nartc** [comment]
@jdegand as for missing readme files, I'd like a separate PR if possible. Thanks!!

**Review by nartc — APPROVED** [review]

**Pull request merged**